src/core/machnet_controller.cc

```diff
@@ -317,15 +320,20 @@ void MachnetController::RunController() {
     this->HandleTimeout(socket);
   };
 
-  server_ = std::make_unique<UDServer>(socket_path, on_connect_cb, on_close_cb,
+  asio::io_context io_context;
```

Comment:
need to wrap 323-327 in a try/catch.

Reply:
We have exceptions explicitly disabled on machnet; is it possible to do it without?